sdk/loadtesting/azure-developer-loadtesting/README.md

```diff
@@ -1,28 +1,29 @@
 # Azure Load Testing client library for Python
+
 Azure Load Testing provides client library in python to the user by which they can interact natively with Azure Load Testing service. Azure Load Testing is a fully managed load-testing service that enables you to generate high-scale load. The service simulates traffic for your applications, regardless of where they're hosted. Developers, testers, and quality assurance (QA) engineers can use it to optimize application performance, scalability, or capacity.
 
 ## Documentation
+
 Various documentation is available to help you get started
 
-<!-- - [Source code][source_code] -->
 - [API reference documentation][api_reference_doc]
 - [Product Documentation][product_documentation]
 
 ## Getting started
 
+### Prequisites
+
+- Python 3.7 or later is required to use this package.
```

Comment:
`setup.py` mentions python 3.8 is required

Reply:
Fixed